streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -1070,6 +1070,10 @@ private Thread shutdownHelper(final boolean error) {
             adminClient.close();
 
             streamsMetrics.removeAllClientLevelMetrics();
```

Comment:
This seems wrong. There are a few duplicate calls here

Reply:
Ahh, got mixed up when I rebased trunk